network-api/networkapi/wagtailpages/pagemodels/customblocks/current_events_slider_list_block.py

```diff
@@ -0,0 +1,39 @@
+from wagtail.core import blocks
+from wagtail.images.blocks import ImageChooserBlock
+
+from .common.link_blocks import LabelledExternalLinkBlock, LabelledInternalLinkBlock
+
+
+class CurrentEventBlock(blocks.StructBlock):
```

Comment:
Could we give this block some meta info so it is labeled in the gui of wagtail 😄